RiotSwiftUI/Modules/Room/TimelinePoll/Coordinator/TimelinePollCoordinator.swift

```diff
@@ -84,8 +84,7 @@ final class TimelinePollCoordinator: Coordinator, Presentable, PollAggregatorDel
     func start() { }
     
     func toPresentable() -> UIViewController {
-        VectorHostingController(rootView: TimelinePollView(viewModel: viewModel.context),
-                                forceZeroSafeAreaInsets: true)
+        VectorHostingController(rootView: TimelinePollView(viewModel: viewModel.context))
```

Comment:
Not saying this is contributing to the issue, but I'm kind of questioning why we create a new hosting controller each time in these `toPresentable`s. The timeline reload a *lot*, and each time it gets a freshly created instance 😕

Reply:
I've questioned that too and tried to use the same instance but it didn't work well at all. So I'll just assume there is a reason for that :)